examples/erc20/DAI/DAI.xml

```diff
@@ -61,18 +126,202 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                <ts:label>
+                    <ts:string xml:lang="en">Amount to supply</ts:string>
+                </ts:label>
+                <ts:origins>
+                    <!-- e18 is a hard coded multiplier.
+                    rationale for hardcoding: avoiding over-design  -->
+                    <ts:user-entry as="e18"/>
+                </ts:origins>
+            </ts:attribute>
+            <ts:transaction>
+                <ethereum:transaction function="mint" contract="cDAI" as="uint">
                     <ts:data>
-                        <ts:address ref="ownerAddress"/>
+                        <ts:uint256 ref="mintAmount"/>
                     </ts:data>
-                </ethereum:call>
+                </ethereum:transaction>
+            </ts:transaction>
+            <ts:view xml:lang="en">
+                <xhtml:style type="text/css">&style;</xhtml:style>
+                <xhtml:script type="text/javascript">&supply.en;</xhtml:script>
+            </ts:view>
+        </ts:card>
+
+        <ts:card type="action" exclude="notEnabled">
+            <ts:label>
+                <ts:string xml:lang="en">Withdraw</ts:string>
+            </ts:label>
+            <ts:attribute name="redeemAmount">
+                <ts:type><ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax></ts:type>
+                <ts:label>
+                    <ts:string xml:lang="en">Amount to withdraw</ts:string>
+                </ts:label>
+                <ts:origins>
+                    <ts:user-entry as="e8"/>
+                </ts:origins>
+            </ts:attribute>
+            <ts:transaction>
+                <ethereum:transaction function="redeem" contract="cDAI" as="uint">
+                    <ts:data>
+                        <ts:uint256 ref="redeemAmount"/>
+                    </ts:data>
+                </ethereum:transaction>
+            </ts:transaction>
+            <ts:view xml:lang="en">
+                <xhtml:style type="text/css">&style;</xhtml:style>
+                <xhtml:script type="text/javascript">&withdraw.en;</xhtml:script>
+            </ts:view>
+        </ts:card>
+
+        <ts:card type="activity" name="enabledForCDAI">
+            <ts:origins>
+                <ethereum:event type="Approval" filter="(&amp;(owner=${ownerAddress})(spender=0x5d3a536e4d6dbd6114cc1ead35777bab948e3643))"/>
+            </ts:origins>
+            <ts:item-view xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+                <style type="text/css">&style;</style>
+                <script  type="text/javascript">&item-view-ownerApproved.en;</script>
+            </ts:item-view>
+            <ts:view xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+                <style type="text/css">&style;</style>
+                <script  type="text/javascript">&item-view-ownerApproved.en;</script>
+            </ts:view>
+        </ts:card>
+
+        <ts:card type="activity" name="mintedCDAI">
+            <ts:origins>
```

Comment:
Have to remember to only bundle this PR in the app and not ship it to the repo since it's the same schema version..

Reply:
Yes